docs/proposals/cluster-templates.md

```diff
@@ -0,0 +1,47 @@
+## Introduction
+
+A cluster template is a reusable cluster template object. The Kubernetes CRD will be used for this purpose.
+A cluster template allows you to specify a provider, node layout, and configurations to materialize a cluster instance
+via Kubermatic API or UI.
+
+During cluster creation or after successful creation the user can store cluster definition as a template.
+
+### Scope
+
+The cluster templates should be accessible from different levels. Should have global (admin), project or user scope.
+Template management will be available from project level.
```

Comment:
Does this mean that the cluster template can not be shared across projects? For example, if a user creates a cluster in one project, then this user cannot use the same template in another project?

Reply:
can be share for the same user in different projects when the template has this user scope. Then the user can use the template in every assigned project